lnrpc/walletrpc/walletkit.proto

```diff
@@ -863,6 +863,76 @@ enum WitnessType {
     transaction.
     */
     COMMITMENT_ANCHOR = 13;
+
+    /*
+    A witness type that is similar to the COMMITMENT_NO_DELAY type,
+    but it omits the tweak that randomizes the key we need to
+    spend with a channel peer supplied set of randomness.
+    */
+    COMMITMENT_NO_DELAY_TWEAKLESS = 14;
```

Comment:
Too bad we added anchor before the tweakless ones... But we can't break the RPC by assigning new values to existing constants, so it makes sense to continue like this, filling the gap but getting out of sync with the `input` package constants...

Reply:
Yeah, since we can't renumber either enum, the second best alternative is just to ensure they are named the same, and ensure all protobuf enum values have a corresponding native enum value.

Reply:
wondering how we can  remove values from the protofile in general ? Is it ever possible to remove a depreciated value completely shifting each following value one position backwards, or would that always break backwards compatibility and we only fill gaps keeping old values as they are ?

Reply:
Unfortunately removing isn't possible. Since protobuf is basically a TLV format where the positional number is the type number. If you shift those, you lose backward compatibility.
What we do to deprecate a value is to replace it with the `reserved` keyword, for example `COMMITMENT_NO_DELAY_TWEAKLESS = 14;` becomes `reserved 14;`.